rotkehlchen/tests/unit/test_assets.py

```diff
@@ -134,6 +134,26 @@ def test_coingecko_identifiers_are_reachable():
         'SGN',
         'SOAR',
         'YUP',
+        'AC',
```

Comment:
Hey @vnavascues when I say to adjust the tokens, I also mean to add here any tokens that appear as delisted in coingecko due to us not being able to find the identifier.

The test shows warning for it, like this:
```
rotkehlchen/tests/unit/test_assets.py::test_coingecko_identifiers_are_reachable
  /home/lefteris/w/rotkehlchen/rotkehlchen/tests/unit/test_assets.py:197: UserWarning: Asset FREC coingecko mapping does not exist.
    test_warnings.warn(UserWarning(msg))
```

The reason to do that is that this can also detect any mistaken coingecko identifiers. Since coingecko website and api **DO NOT** always have the same identifiers. It's pretty messed up I know.

```
rotkehlchen/tests/unit/test_assets.py::test_coingecko_identifiers_are_reachable
  /home/lefteris/w/rotkehlchen/rotkehlchen/tests/unit/test_assets.py:197: UserWarning: Asset FRONT coingecko mapping does not exist.
  Suggestion: id:frontier-token name:Frontier symbol:front
    test_warnings.warn(UserWarning(msg))

```

But thanks to this test I found out that:

`bitcoin-cash-abc-2`, `frontier-token` and  `sharering` had wrong coingecko identifier and adjusted them. You can see it in this commit.